### PR TITLE
Added README requirement to Definition of Done

### PR DESCRIPTION
**Pull request by amyjohnston**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -16,20 +16,22 @@ This section describes how to determine whether a blueprint has enough value to
 This section describes how to determine whether the blueprint has the right technical quality to be included in the central XebiaLabs blueprints repository.
 
 - [ ] The branch from which the PR is created is up-to-date with the `development` branch.
+- [ ] There is a `README.md` file at the root of the blueprint folder that describes the blueprint, using the standard README template.
```
Comment (vpartington) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is already in the DOD. See the next line. :-)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -16,20 +16,22 @@ This section describes how to determine whether a blueprint has enough value to
...
- 
+
+
 **Review and testing**
 
 This section describes how to determine whether the blueprint has been reviewed and tested well enough to be included in the central XebiaLabs blueprints repository.
 
 - [ ] The CI for the PR is green
 - [ ] The blueprint has been reviewed by someone else in the team.
+- [ ] The blueprint README has been reviewed by a Technical Writer and a Product Marketing Manager,.
```
Comment (vpartington) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do those titles include upper case letters? ;-)

Comment (amyjohnston) [reply .github/PULL_REQUEST_TEMPLATE.md]: OK, maybe it's overkill. 😁

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (amyjohnston) [reply .github/PULL_REQUEST_TEMPLATE.md]: I took a shot at merging the two. Haven't had a chance to chat with Deepu about it yet.

**Comment by amyjohnston** [comment]
> These links are old ones. Should minimally use these:
> 
> https://docs.xebialabs.com/xl-platform/concept/get-started-with-devops-as-code.html (install CLI and CLI reference are linked from this one)
> https://docs.xebialabs.com/xl-platform/concept/get-started-with-blueprints.html

Fixed!

**Pull request merged**